.github/workflows/docker.yml

```diff
@@ -9,7 +9,7 @@ jobs:
     steps:
       - uses: actions/checkout@v2
       - name: build
-        uses: docker/build-push-action@v1.1.1
+        uses: docker/build-push-action@v2.0.1
```

Comment:
tests pass because we do not execute the publishing from branches, but this will break publishing.

this is what's needed to upgrade: https://github.com/docker/build-push-action/blob/master/UPGRADE.md#simple-workflow